<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>771</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Statistics Calculator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>761</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="input_label">
       <property name="text">
        <string>Input data:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="saveData_button">
       <property name="text">
        <string>Load data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearData_button">
       <property name="text">
        <string>Clear all data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="count_label">
       <property name="text">
        <string>Data count:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="countDisplay_lcd">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Oct</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>271</width>
      <height>157</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="arithmeticMean_checkBox">
           <property name="text">
            <string>Arithmetic Mean</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="median_checkBox">
           <property name="text">
            <string>Median</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="minimum_checkBox">
           <property name="text">
            <string>Minimum</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="maximum_checkBox">
           <property name="text">
            <string>Maximum</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLCDNumber" name="arithmeticMean_lcd"/>
         </item>
         <item>
          <widget class="QLCDNumber" name="median_lcd"/>
         </item>
         <item>
          <widget class="QLCDNumber" name="minimum_lcd"/>
         </item>
         <item>
          <widget class="QLCDNumber" name="maximum_lcd"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="displayData_button">
       <property name="text">
        <string>Display selected data</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
